woocommerce/woocommerce-ios
# Replace `ImageAndTitleAndTextTableViewCell.TopLeftImageViewModel` with `imageAndTitleOnly` style

Part of #3419 

## Why

Similar to https://github.com/woocommerce/woocommerce-ios/pull/3470, we are replacing `ImageAndTitleAndTextTableViewCell`'s view models with `Style` enum. `imageAndTitleOnly` style case has been used in the app, and this PR replaced the last usage of `ImageAndTitleAndTextTableViewCell.TopLeftImageViewModel` with `imageAndTitleOnly` style in order details > shipping notice.

## Changes

- Removed manual tint color for `UIImage.noticeImage`, otherwise the icon cannot change tint color after switching light/dark mode
- In `OrderDetailsDataSource.configureShippingNotice`, replaced `ImageAndTitleAndTextTableViewCell.TopLeftImageViewModel` with `imageAndTitleOnly` style configuration
- In `ImageAndTitleAndTextTableViewCell`, removed `TopLeftImageViewModel` code

## Testing

The conditions when the shipping notice is shown require some work (the order needs to have multiple shipping methods), and I recommend commenting out these lines so that the shipping notice section is always shown in order details:

https://github.com/woocommerce/woocommerce-ios/blob/42402d6b6799e24eb5b5d832476cb712fe2f7f2d/WooCommerce/Classes/ViewModels/Order%20Details/OrderDetailsDataSource.swift#L727-L731

- Go to the Orders tab
- Tap on an order where the shipping notice section is shown --> the shipping notice section should look as before

## Example screenshots

(No changes are expected)

before | after
-- | --
![Simulator Screen Shot - iPhone 11 - 2021-01-14 at 16 02 01](https://user-images.githubusercontent.com/1945542/104565212-e99e3c80-5686-11eb-9916-fbc5758e760b.png) | ![Simulator Screen Shot - iPhone 11 - 2021-01-14 at 16 10 37](https://user-images.githubusercontent.com/1945542/104565090-b8be0780-5686-11eb-81ea-d1bac6b50c5a.png)
![Simulator Screen Shot - iPhone 11 - 2021-01-14 at 16 04 32](https://user-images.githubusercontent.com/1945542/104565059-ac39af00-5686-11eb-8ea2-18ff108db190.png) | ![Simulator Screen Shot - iPhone 11 - 2021-01-14 at 16 10 59](https://user-images.githubusercontent.com/1945542/104565151-cf645e80-5686-11eb-8454-f2cc4e7c0d79.png)


Update release notes:
